Makefile

```diff
@@ -59,6 +60,9 @@ COMPONENTS_USAGE_STATS_FILES = $(shell \
 		-type f \
 		\( -name '*.js' -or -name '*.jsx' \) \
 )
+COMPONENTS_PROPTYPE_FILES = $(shell \
+	find client -name 'index.jsx' -or -name 'example.jsx' \
```

Comment:
> There's some components in the devdocs that are outside of the components and blocks folders

Indeed. But those examples should probably be [blocks](https://github.com/Automattic/wp-calypso/tree/master/client/blocks#blocks) — apart from `sites-list`, which isn't a component at all, and should not exist :)